packages/kubernetes-1.21/kubelet-config

```diff
@@ -113,7 +113,12 @@ featureGates:
   CSIMigration: false
 protectKernelDefaults: true
 serializeImagePulls: false
+{{#if (and (default "" settings.kubernetes.tls-certificate) (default "" settings.kubernetes.tls-private-key))}}
```

Comment:
I'm not too happy about this, but without the `default`s in there the rendering of the file blows up with a handlebars error:

```
helper: Couldn't read parameter x
```

If anyone knows of a better way, please let me know.

Reply:
We would still need the `if` conditional to determine whether to write out of the `tlsCert` and `tlsPrivateKeyFile` though, right? So the double `unless` would work to determine if we need to write out `serverTLSBootstrap`, but we also need to know when to do the inverse.